src/lib/network/network_autoyast.rb

```diff
@@ -105,5 +190,17 @@ def merge_routing(instsys_routing, ay_routing)
 
       instsys_routing.merge(ay_routing)
     end
+
+    # Returns networking section of current AY profile
+    def ay_networking_section
+      Yast.import "Profile"
+
+      ay_profile = Profile.current
+
+      return {} if ay_profile.nil? || ay_profile.empty?
+      return {} if ay_profile["networking"].nil?
+
+      ay_profile["networking"]
+    end
```

Comment:
bcs I'm simple and i want to see explicitly what is return value in edge cases ... just say coding style.